common/lib/xmodule/xmodule/tests/test_xblock_wrappers.py

```diff
@@ -387,6 +388,21 @@ def test_xmodule_handler_return_value(self):
         self.assertIsInstance(response, webob.Response)
         self.assertEqual(response.body, '{}')
 
+    @ddt.data(
+        u'{"test_key": "test_value"}',
+        '{"test_key": "test_value"}',
+    )
+    def test_xmodule_handler_with_data(self, response_data):
+        """
+        Webob Response (v1.7) raises an error for application/json content-type that does not have a charset.
+        Tests that webob Response does not raise an error if charset is provided.
```

Comment:
Tests that xmodule_handler function correctly wraps handle_ajax does when handle_ajax response is either `str` or `uniocde`.